dev/moving_providers/move_providers.py

```diff
@@ -357,10 +357,13 @@ def move_provider_yaml(provider_id: str) -> tuple[list[str], list[str], list[str
     dependencies = []
     optional_dependencies = []
     devel_dependencies = []
+    copied_logo = set()
```

Comment:
fix `FileNotFoundException` when there are multiple logos in the `integrations` section.